weldx/asdf/tags/weldx/measurement/signal_transformation.py

```diff
@@ -0,0 +1,24 @@
+from weldx.asdf.types import WeldxType
+from weldx.measurement import SignalTransformation
+
+__all__ = ["SignalTransformation", "SignalTransformationType"]
+
+
+class SignalTransformationType(WeldxType):
+    """Serialization class for data transformations."""
+
+    name = "measurement/signal_transformation"
+    version = "1.0.0"
+    types = [SignalTransformation]
+    requires = ["weldx"]
+    handle_dynamic_subclasses = True
+
+    @classmethod
+    def to_tree(cls, node: SignalTransformation, ctx):
+        """convert to tagged tree and remove all None entries from node dictionary"""
```

Comment:
```suggestion
```
This is outdated anyway so we can remove those docstrings in other functions as well

if we haven't already I'm fine with deactivating pydocstyle for all `to_tree` and `from_tree` methods since it's pretty clear what they do